x-pack/platform/plugins/private/translations/translations/fr-FR.json

```diff
@@ -16458,7 +16456,6 @@
     "xpack.elasticAssistant.knowledgeBase.tour.videoStep.title": "Présentation des sources de connaissances personnalisées",
     "xpack.elasticAssistant.prompts.bulkActionspromptsError": "Erreur de la mise à jour des invites {error}",
     "xpack.elasticAssistant.prompts.getPromptsError": "Erreur lors de la récupération des invites",
-    "xpack.elasticAssistant.securityAssistant.content.prompts.welcome.enterprisePrompt": "L'assistant d'IA d'Elastic n'est accessible qu'aux entreprises. Veuillez mettre votre licence à niveau pour bénéficier de cette fonctionnalité.",
```

Comment:
Curious how these translations are sourced? I know this is being removed but this was wrong.

Reply:
@lukeelmers FYI, in case this weighs on how we source our translations.